src/nvim/options.lua

```diff
@@ -3368,6 +3368,8 @@ return {
         Format to recognize for the ":grep" command output.
         This is a scanf-like string that uses the same format as the
         'errorformat' option: see |errorformat|.
+
+        If ripgrep (|'grepprg'|) is available, this option defaults to `%f:%l:%c:%m`.
```

Comment:
single-quotes automatically link options, so bars are redundant.
```suggestion
        If ripgrep ('grepprg') is available, this option defaults to "%f:%l:%c:%m".
```

Reply:
done